src/vs/workbench/contrib/search/browser/searchResultsView.ts

```diff
@@ -401,10 +401,10 @@ export class SearchAccessibilityProvider implements IListAccessibilityProvider<R
 			const range = match.range();
 			const matchText = match.text().substr(0, range.endColumn + 150);
 			if (replace) {
-				return nls.localize('replacePreviewResultAria', "Replace '{0}' with '{1}' in line '{2}' at column {3}", matchString, match.replaceString, matchText, range.startColumn + 1);
+				return nls.localize('replacePreviewResultAria', "'{0}' at column {1} replace with {2}", matchText, range.startColumn + 1, matchString);
```

Comment:
doesn't this say `replace with` and then the original string? Shouldn't it be `match.replaceString` instead of matchString? Also, I'm a little hesitant about losing the original string from the aria, since it gives info on what's being replaced.

Reply:
yes that should be `match.replaceString`. if you would like to keep that part, we can for sure. Just need to append it to the end so the important content is read first.

Reply:
yes, we could probably do something like `replace x with y` instead of `replace with y` so that it's more clear